src/utils/print-number.js

```diff
@@ -1,4 +1,10 @@
 function printNumber(rawNumber) {
+  if (rawNumber.length === 1) {
+    const code = rawNumber.charCodeAt(0);
+    if (code >= /* "1" */ 48 && code <= /* "9" */ 57) {
```

Comment:
Did you mean `/* "0" */`?

But what's the point of this check, what possible single character can it be?

Reply:
[34eba8a](https://github.com/prettier/prettier/pull/16034/commits/34eba8aabff0a93b61ac68fa535adba93dab5603)